sklearn/metrics/ranking.py

```diff
@@ -47,12 +47,23 @@ def auc(x, y, reorder=False):
     Parameters
     ----------
     x : array, shape = [n]
-        x coordinates.
+        x coordinates. These must be either monotonic increasing or monotonic
+        decreasing.
     y : array, shape = [n]
         y coordinates.
-    reorder : boolean, optional (default=False)
-        If True, assume that the curve is ascending in the case of ties, as for
-        an ROC curve. If the curve is non-ascending, the result will be wrong.
+    reorder : boolean, optional (default='deprecated')
+        Whether to sort x before computing. If False, assume that x must be
+        either monotonic increasing or monotonic decreasing. If True, y is
+        used to break ties when sorting x. Make sure that y has a monotonic
+        relation to x when setting reorder to True.
+
+        ..deprecated:: 0.20
+          Parameter ``reorder`` has been deprecated in version 0.20 and will
+          be removed in 0.22. It's introduced for roc_auc_score (not for
+          general use) and is no longer used there. What's more, the result
+          from auc will be significantly influenced if x is sorted
+          unexpectedly due to slight floating point error (See issue #9786).
+          Future (and default) behavior is equivalent to `reorder=False`.
```

Comment:
This is rst and not markdown so you need double-backticks around reorder=False. Single backtick is italic in rst.